components/cli/pkg/commands/login.go

```diff
@@ -101,3 +125,158 @@ func RunLogin(registryURL string, username string, password string) {
 	spinner.Stop(true)
 	util.PrintSuccessMessage(fmt.Sprintf("Successfully logged into Registry: %s", util.Bold(registryURL)))
 }
+
+// This function is to request credentials from the user
+func requestCredentials(username string) (string, string, error) {
+	initConf()
+	timeout := make(chan bool)
+	ch := make(chan string)
+	var code string
+	httpPortString := ":" + strconv.Itoa(loginConf.CallBackDefaultPort)
+	var codeReceiverPort = loginConf.CallBackDefaultPort
+	// This is to start the CLI auth in a different port is the default port is already occupied
+	for {
+		_, err := net.Dial("tcp", httpPortString)
+		if err != nil {
+			break
+		}
+		codeReceiverPort++
+		httpPortString = ":" + strconv.Itoa(codeReceiverPort)
+	}
+	var gitHubAuthUrl = "https://" + loginConf.IsHost + ":" + strconv.Itoa(loginConf.IsPort) +
+		"/oauth2/authorize?scope=openid&response_type=" + "code&redirect_uri=http%3A%2F%2F" +
+		loginConf.CallBackHost + "%3A" + strconv.Itoa(codeReceiverPort) +
+		"%2Fauth&client_id=" + loginConf.SpClientId + "&fidp=google"
+	fmt.Println("\n", gitHubAuthUrl)
```

Comment:
Shouldn't this print be removed ?

Reply:
I should be included.. Since we are logging the url like in gcloud